.cirrus.yml

```diff
@@ -33,7 +33,31 @@ win_vm_definition: &WINDOWS_VM_DEFINITION
     type: n1-standard-8
     use_ssd: true
 
+generate_typeshed_task:
```

Comment:
With this approach, aren't we building `python-frontend` twice when there are changes to the serializer? Once with the `generateTypeshed` flag, and once without?

I guess we don't risk overriding anything but maybe we could also have the build script setting `generateTypeshed` conditionally depending on the changes, so that there is no duplication? One counter argument could be that then it's less visible whether Typeshed has been generated or not (though maybe we could simply have a log for that?)

Reply:
As discussed, it looks like we could only call `changesInclude` inside values of `skip` and `only_if` properties. Going to keep it as a separate task for the time being